app/src/main/res/values/strings.xml

```diff
@@ -1763,4 +1763,10 @@
     <string name="pocket_stories_feature_caption">Part of the Firefox family. %s</string>
     <!-- Clickable text for opening an external link for more information about Pocket. -->
     <string name="pocket_stories_feature_learn_more">Learn more</string>
+
+    <!-- Compose components -->
+    <!-- Content description for expanding a section of items in a list. -->
+    <string name="expand_list_section">Expand list section</string>
+    <!-- Content description for collapsing a section of items in a list. -->
+    <string name="collapse_list_section">Collapse list section</string>
```

Comment:
I am trying to think if this is too generic for content description and if we should be providing a param for content description. The obvious example against having these generic content description is "Collapse group of synced tabs" and "Expand group of synced tabs".